Target.java

```diff
@@ -1,17 +1,59 @@
-/*
- * File: Target.java
- * Name: 
- * Section Leader: 
- * -----------------
- * This file is the starter file for the Target problem.
- */
-
-import acm.graphics.*;
-import acm.program.*;
-import java.awt.*;
-
-public class Target extends GraphicsProgram {	
-	public void run() {
-		/* You fill this in. */
-	}
-}
+/*
+ * File: Target.java
+ * Name: 
+ * Section Leader: 
+ * -----------------
+ * This file is the starter file for the Target problem.
+ */
+
+import acm.graphics.*;
+import acm.program.*;
+import java.awt.*;
+
+public class Target extends GraphicsProgram {
+	
+	
+	//
+	double InchesToRadius = 72;
+	double outerRad = 1.0 * InchesToRadius;
+	double middleRad = 0.65 * InchesToRadius;
+	double innerRad = 0.3 * InchesToRadius;
+	double centerX = getWidth()/2;
+	double centerY = getHeight()/2;
+	

```

Comment:
The values for the center (centerX and centerY) should be calculated within the run method instead of here. Also constants are declared as follows:
```
	private static final int INCHES_TO_RADIUS = 72;
	private static final double OUTER_RAD = 1.0 * INCHES_TO_RADIUS;
	private static final double MIDDLE_RAD = 0.65 * INCHES_TO_RADIUS;
	private static final double INNER_RAD = 0.3 * INCHES_TO_RADIUS;
```